webpack.common.config.ts

```diff
@@ -678,6 +678,9 @@ module.exports = {
             // Typescript checks do the type checking.
             transpileOnly: true
           }
+        },
+        {
+          loader: 'angular2-template-loader'
```

Comment:
I'm very poor at webpack things -- what is thing doing? 

(Allowing us to use templateUrl:()? )

Reply:
Yes, and these are the two lines I missed in the pr I made to your branch.